lib/app_revision.rb

```diff
@@ -0,0 +1,15 @@
+class AppRevision
```

Comment:
I see it as a utility class and not part of the data model. Also don't want it to be confused with gem Version.